### DISCUSSION TITLE
🏈 Remastered (Early Beta)

### DISCUSSION THREAD

**Discussion by swiftyfinch** [swiftyfinch/Rugby#205] [Releases]
### 🙋🏼‍♀️ Hello everybody!
I’m glad to share great news with you.
Finally, I have almost finished working on the new version.
I called it 🏈 <b>Rugby: Remastered</b> 😀.

### 🚀 Welcome to Early Public Beta!
Now, the new version hasn't got all features from the Rugby `1.x`, but it is already stable enough.
I have been using it in production development since October.

Please, check out the [readme](https://github.com/swiftyfinch/Rugby/tree/2.0.0b2#readme). You can find there all information.

###### 🎬 Demo

<img src="https://user-images.githubusercontent.com/64660122/209442704-3f636690-0340-46b5-a3b9-1d9e3779ba82.gif" width=400>

<hr /><em>This discussion was created from the release <a href='https://github.com/swiftyfinch/Rugby/releases/tag/2.0.0b2'>🏈 Remastered (Early Beta)</a>.</em>

**Comment by wei18** [comment]
Congrats, lol.
